### ISSUE TITLE
[proposal] [ux] Make kernel_size = 1 default argument value for convolutions

### ISSUE THREAD

**Issue by vadimkantorov** [pytorch/pytorch#57535]
Since, currently Linear does not support dim (https://github.com/pytorch/pytorch/issues/52143), we may have to change from nn.Linear to nn.Conv*d to do pointwise convolutions. 

Currently: `nn.Linear(in_dim, out_dim) -> nn.Conv1d(in_dim, out_dim, 1)`
With change: `nn.Linear(in_dim, out_dim) -> nn.Conv1d(in_dim, out_dim)`

cc @albanD @mruberry @jbschlosser

**Comment by mrshenli** [comment]
Adding "triage review" tag to discuss this one in triage review meeting

**Comment by albanD** [comment]
Hey,

We had a chat about that and we don't think this is a good feature to add.
The main reason is that while it can be useful in that context, it is not a good default in general. And in particular, the error message that people will get if they forget to pass in the kernel size would be much worst (a weird shape error during the forward instead of a clear Error at the Module creation).

That being said, we agree that it can be useful in this case but hopefully a simple Conv1d wrapper with the extra default should be easy to do for this use case.
